src/DesktopTests/UIAutomation/CustomObjects/RegistrarTests.cs

```diff
@@ -96,5 +96,44 @@ public void CreateTypeConverterUnsetTest()
         {
             Assert.ThrowsException<ArgumentException>(() => Registrar.CreateTypeConverter(new CustomProperty()));
         }
+
+        [TestMethod, Timeout(1000)]
+        public void MergeRestorePropertyTest()
+        {
+            Mock<IUIAutomationRegistrar> uiaRegistrarMock = new Mock<IUIAutomationRegistrar>(MockBehavior.Strict);
+
+            Guid intPropertyGuid = new Guid("312F7536-259A-47C7-B192-AA16352522C4");
```

Comment:
Should we use a property that isn't defined by any known app? It probably doesn't matter that much, but it seems odd to "borrow" one of Excel's properties

Reply:
We do this in other registrar tests...